auto_nag/scripts/vacant_triage_owner.py

```diff
@@ -0,0 +1,167 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from jinja2 import Environment, FileSystemLoader
+from libmozdata.bugzilla import BugzillaProduct
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag.nag_me import Nag
+from auto_nag.team_managers import TeamManagers
+from auto_nag.user_activity import UserActivity
+
+
+class TriageOwnerVacant(BzCleaner, Nag):
+    def __init__(self):
+        super(TriageOwnerVacant, self).__init__()
+        self.query_url = None
+
+    def description(self):
+        return "Components with triage owner need to be assigned"
+
+    def fetch_products(self):
+        data = []
+        include_fields = [
+            "name",
+            "is_active",
+            "components.id",
+            "components.name",
+            "components.team_name",
+            "components.triage_owner",
+            "components.is_active",
+        ]
+
+        def product_handler(product, data):
+            data.append(product)
+
+        BugzillaProduct(
+            product_names=self.get_products(),
+            include_fields=include_fields,
+            product_types=["accessible", "selectable", "enterable"],
```

Comment:
`accessible` should be enough, as it should contain both `selectable` and `enterable` according to https://bugzilla.readthedocs.io/en/latest/api/core/v1/product.html

Reply:
I checked the docs, but I could not find where it mentions that `accessible` does include `selectable` and `enterable`. Anyway, I think we can remove `product_types` since `accessible` is already the default value. Also, since we specify the product names, I think the `product_types` field is useless.

Reply:
Done in ce1444a98bd29331b13c067ac577fdedb92c3e9d